app/parse.py

```diff
@@ -14,5 +21,34 @@ class Course:
     course_type: CourseType
 
 
+def parce_single_course(course_soup: Tag) -> Course:
+    course = Course(
+        name=course_soup.select_one(
+            ".typography_landingH3__vTjok"
+        ).text,
+        short_description=course_soup.select_one(
+            ".typography_landingP1__N9PXd"
+        ).text,
+        course_type=CourseType.FULL_TIME
+    )
+    if course.name.endswith("Вечерний"):
+        course.course_type = CourseType.PART_TIME
```

Comment:
```
course_type = (
    CourseType.PART_TIME
    if course.name.endswith("Вечерний") else
    CourseType.FULL_TIME
)

```

Reply:
And after create object
No need to change field better to create in correct way